lib/klaviyo/apis/templates.rb

```diff
@@ -0,0 +1,105 @@
+module Klaviyo
+  class Templates < Client
+    TEMPLATES = 'email-templates'
+    CLONE = 'clone'
+    RENDER = 'render'
+    SEND = 'send'
+
+    # Retrieve all email templates available.
+    # @param person_id [String] the id of the profile
+    # @return [JSON] A dictionary with a data property that contains an array of all the
+    #   email templates. Each entry is a separate email template object. If no email templates exist,
+    #   the resulting array will be empty. This request should never return an error.
+    def self.get_templates
+      v2_request(HTTP_GET, TEMPLATES)
+    end
+
+    # Creates a new email templates
```

Comment:
Small nit: template should be singular here, String is misspelled on the following line